fix(ingest/powerbi): support each expression in m-query function invocation (#7541)
datahub-project/datahub: metadata-ingestion/docs/sources/powerbi/powerbi_pre.md

`Pattern-1` is supported as it first assign the table from schema to variable and then variable is used in M-Query Table function i.e. Table.Combine

> Note: If you are using (PowerBI Parameters)[https://learn.microsoft.com/en-us/power-query/power-query-query-parameters]. then avoid using same name for variable in M-Query

Reviewer: we should also add the caveat that we don't support parameters with the admin API

Author: Added it in `Admin Ingestion vs. Basic Ingestion`

  - Dashboard's Tiles
  - Report's Pages

> Lets consider user don't want (or doesn't have access) to add service principal as member in workspace then you can enable the `admin_apis_only: true` in recipe to use PowerBI Admin API only. 

Reviewer: ```suggestion
If you don't want to add a service principal as a member in your workspace, then you can enable the `admin_apis_only: true` in recipe to use PowerBI Admin API only. 
```

> if `admin_apis_only` is set to `true` then following metadata would not get ingested 

Reviewer: ```suggestion
Caveats of setting `admin_apis_only` to `true`:
```

Author: done
